### ISSUE TITLE
Ank fails to generate hierarchy for typeclasses that don't inherit from another typeclass

### ISSUE THREAD

**Issue by 1Jajen1** [arrow-kt/arrow#1164]
Currently (before I refractored ank a bit and after as well) the ank macro fails to generate a diagram for typeclasses like eq, show etc. Also is it supposed to not show methods anymore?

**Comment by 1Jajen1** [comment]
Ah ok, I think it's fine, only confusing because it always generates `## Type Class Hierarchy` regardless of a diagram actually being there or not. The methods did take a lot of space so not showing them seems better.

**Comment by raulraja** [comment]
yeah the generation of that is a bug, we need to remove the headers if the extends list is empty.
